NEWS

```diff
@@ -31,27 +47,31 @@ New Functionality
   available in the Zeek documentation. Note also that Spicy is currently
   unsupported and will be fixed in the future.
 
+  The feature as checked into the repository is not considered production-ready.
```

Comment:
When reading "the feature" I was wondering about something specific/small, but that's about Windows support.

Maybe: "Windows is not considered a production-ready platform at this point."